_data/team/roles/outreach.yml

```diff
@@ -0,0 +1,32 @@
+role: Outreach
+subroles:
+  - subrole: Workshops and meetings
+    tasks:
+    - Identifying workshop and event opportunities
+    - Organising workshops and events
+    - Gathering and coordinating volunteers
+    - Managing content, presentations and teaching at workshops
+    current_leads:
+      - Jenna Swarthout Goddard
+      - Rocco Meli
```

Comment:
```suggestion
```

I have a keen interest in teaching, but my current work and life situation makes it difficult to participate to/teach at workshops, both in person and online. Happy to re-evaluate in the near future, but I'm not sure it makes a lot of sense for me to lead this right now.

In the original Google Sheet, I added myself as lead of the sub-task/sub-role "Material still works / fix as needed" which is not reflected here. What I had in mind when filling the spreadsheet, is work on the lines of https://github.com/MDAnalysis/WorkshopMDMLEdinburgh2022/pull/15, that I can do in my spare time without having to commit to particular time slots (which is a bit difficult for me right now). Something like the "Manage material maintenance on GitHub" task below, for mentoring.

Reply:
Ah I see, thanks for the clarification @RMeli. Would it make sense and would you be happy for me to split out "material maintenance" as a separate subrole in "Outreach", separate to workshops and mentoring? Teaching materials are somewhat different than the non-core package maintenance that already exists.

Reply:
Yes, thank you for adding it!